wesnoth-units.po

```diff
@@ -2533,13 +2541,41 @@ msgid ""
 "the other Wayfarers again. My tale might not be that impressive to them, but "
 "at least it is an improvement over my previous encounter with the wyvern...”"
 msgstr ""
+"Z deníku Balaha, pouštního strážce:\n"
+"\n"
+"„... Dnes to byl šestý den. Šest dní pronásledování skrz poušť bez spánku a "
+"bez odpočinku... ale nevyžaduji ani jedno. Rudooký Roh ukradl z mého stáda "
+"příliš mnoho kusů pro své dravé hodování. Musím se ho zbavit, než karavana "
+"vyhladoví. Sledoval jsem ho tak daleko. A bez ohledu na to, jak daleko "
+"uprchne, ho budu sledovat dál.“\n"
```

Comment:
nenašel jsem, že by čeština na konci odstavce ukončovala uvozovky; originál to také nemá
```suggestion
"uprchne, ho budu sledovat dál.\n"
```